git_machete/github.py

```diff
@@ -180,8 +180,10 @@ def __fire_github_api_request(method: str, path: str, token: Optional[str], requ
                                  'Visit `https://github.com/settings/tokens` to generate a new one.')
         elif err.code == http.HTTPStatus.NOT_FOUND:
             raise MacheteException(
-                f'Given endpoint: {url}, not found in GitHub or you don\'t have'
-                f' the permission to access it (expired token?).')  # TODO (#164): make dedicated exception here
+                f'Given endpoint: {url}, not found in GitHub. You don\'t have '
```

Comment:
Pls also apply the improvements suggested in https://github.com/VirtusLab/git-machete/issues/506#issuecomment-1142068932